### PR TITLE
Marking required fields in SSO and ldap

### PR DESCRIPTION
**Pull request by bmozaffa**
Marking required fields in SSO and ldap

Signed-off-by: Babak Mozaffari <bmozaffa@redhat.com>

### CODE REVIEW

File: frontend/package-lock.json [hunk 1]
```diff
@@ -5,9 +5,9 @@
   "requires": true,
   "dependencies": {
     "@babel/cli": {
-      "version": "7.4.4",
-      "resolved": "https://registry.npmjs.org/@babel/cli/-/cli-7.4.4.tgz",
-      "integrity": "sha512-XGr5YjQSjgTa6OzQZY57FAJsdeVSAKR/u/KA5exWIz66IKtv/zXtHy+fIZcMry/EgYegwuHE7vzGnrFhjdIAsQ==",
+      "version": "7.4.3",
+      "resolved": "https://registry.npmjs.org/@babel/cli/-/cli-7.4.3.tgz",
+      "integrity": "sha512-cbC5H9iTDV9H7sMxK5rUm18UbdVPNTPqgdzmQAkOUP3YLysgDWLZaysVAfylK49rgTlzL01a6tXyq9rCb3yLhQ==",
```
Comment (bmozaffa) [inline frontend/package-lock.json]: @swati-kale @ruromero Maybe before merging this, you can help me figure out why my build downgrades these components?

Comment (ruromero) [reply frontend/package-lock.json]: Don't worry about this for now. Babel-cli and all the babel related dependencies should be moved to devDependencies. I will take care of that in my next PR.
The change of version might be related to your npm version

Comment (ruromero) [reply frontend/package-lock.json]: This is what I have
```
↳ npm version
{ 'operator-gui': '0.1.0',
  npm: '6.9.0',
  ares: '1.14.0',
  cldr: '33.1',
  http_parser: '2.8.0',
  icu: '62.1',
  modules: '64',
  napi: '3',
  nghttp2: '1.34.0',
  node: '10.13.0',
  openssl: '1.1.0i',
  tz: '2018e',
  unicode: '11.0',
  uv: '1.23.2',
  v8: '6.8.275.32-node.36',
  zlib: '1.2.11' }
```

Comment (ruromero) [reply frontend/package-lock.json]: Can you however, not commit this file?

Comment (bmozaffa) [reply frontend/package-lock.json]: The weird thing is that mine are all newer versions, yet somehow result in picking up older versions of the dependencies.

I could manually skip this file, but isn't it resulting in scripts bundled into my packr files?

Comment (bmozaffa) [reply frontend/package-lock.json]: Committing this without the package-lock, though it's presumably impacting the packr file anyway

**Pull request merged**